sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -197,49 +246,28 @@ async def create_queue(self, queue: Union[str, QueueDescription], **kwargs) -> Q
             entry_ele = cast(
                 ElementTree,
                 await self._impl.entity.put(
-                    queue_name,  # type: ignore
+                    name,  # type: ignore
                     request_body, api_version=constants.API_VERSION, **kwargs)
             )
 
         entry = QueueDescriptionEntry.deserialize(entry_ele)
-        result = QueueDescription._from_internal_entity(queue_name,
+        result = QueueProperties._from_internal_entity(name,
             entry.content.queue_description)
         return result
 
-    async def update_queue(
-            self, queue: QueueDescription,
-            *,
-            default_message_time_to_live: timedelta = None,
-            lock_duration: timedelta = None,
-            dead_lettering_on_message_expiration: bool = None,
-            duplicate_detection_history_time_window: timedelta = None,
-            max_delivery_count: int = None,
-            **kwargs) -> None:
+    async def update_queue(self, queue: QueueProperties, **kwargs) -> None:
         """Update a queue.
 
-        :param queue: The properties of this `QueueDescription` will be applied to the queue in
-         ServiceBus. Only a portion of properties can be updated.
-         Refer to https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue.
-        :type queue: ~azure.servicebus.management.QueueDescription
-        :keyword timedelta default_message_time_to_live: The value you want to update to.
-        :keyword timedelta lock_duration: The value you want to update to.
-        :keyword bool dead_lettering_on_message_expiration: The value you want to update to.
-        :keyword timedelta duplicate_detection_history_time_window: The value you want to update to.
-        :keyword int max_delivery_count: The value you want to update to.
+        Before calling this method, you should use `get_queue` to get a `QueueProperties` instance, then update
+        the properties you want to update. Only a portion of properties can be updated.
+        Refer to https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue.
+
+        :param queue: The queue that is returned from `get_queue` and has the updated properties.
```

Comment:
can the "QueueProperties" returned from `create_queue` be used to update queue?

Reply:
I would hope/assume the answer is yes across the board?  At the end of the day the goal being "only able to update if you have the artifact representing a queue-that-already-exists and is thusly fully populated.

Reply:
Updated the docstring to add create_queue and list_queues. Also added same thing to other entities.